api/tests/unit/domain/services/token-service_test.js

```diff
@@ -24,100 +27,94 @@ describe('Unit | Domain | Service | Token Service', function() {
       };
 
       // when
-      const token = tokenService.createIdTokenForUserReconciliation(externalUser);
+      const idToken = tokenService.createIdTokenForUserReconciliation(externalUser);
 
       // then
-      const decodedToken = jsonwebtoken.verify(token, settings.authentication.secret);
-      expect(_.omit(decodedToken, ['iat', 'exp'])).to.deep.equal(expectedTokenAttributes);
+      const decodedToken = jsonwebtoken.verify(idToken, settings.authentication.secret);
+      expect(omit(decodedToken, ['iat', 'exp'])).to.deep.equal(expectedTokenAttributes);
     });
   });
 
   describe('#extractExternalUserFromIdToken', () => {
 
-    it('should return external user if the id token is valid', async () => {
+    it('should return external user if the idToken is valid', async () => {
       // given
       const externalUser = {
         firstName: 'Saml',
         lastName: 'Jackson',
         samlId: 'SamlId',
       };
 
-      const token = tokenService.createIdTokenForUserReconciliation(externalUser);
+      const idToken = tokenService.createIdTokenForUserReconciliation(externalUser);
 
       // when
-      const result = await tokenService.extractExternalUserFromIdToken(token);
+      const result = await tokenService.extractExternalUserFromIdToken(idToken);
 
       // then
       expect(result).to.deep.equal(externalUser);
     });
 
-    it('should throw an InvalidTemporaryKeyError if the token is invalid', async () => {
+    it('should throw an InvalidTemporaryKeyError if the accessToken is invalid', async () => {
       // given
-      const token = 'eyJhbGciOiJIUzI1NiIsIgR5cCI6IkpXVCJ9.eyJ1c2VyX2lPIjoxMjMsImlhdCI6MTQ5OTA3Nzg2Mn0.FRAAoowTA8Bc6BOzD7wWh2viVN47VrPcGgLuHi_NmKw';
+      const accessToken = 'eyJhbGciOiJIUzI1NiIsIgR5cCI6IkpXVCJ9.eyJ1c2VyX2lPIjoxMjMsImlhdCI6MTQ5OTA3Nzg2Mn0.FRAAoowTA8Bc6BOzD7wWh2viVN47VrPcGgLuHi_NmKw';
 
       // when
-      const error = await catchErr(tokenService.extractExternalUserFromIdToken)(token);
+      const error = await catchErr(tokenService.extractExternalUserFromIdToken)(accessToken);
       expect(error).to.be.an.instanceof(InvalidTemporaryKeyError);
```

Comment:
Je propose un BSR de renommer le InvalidTemporaryKeyError en modifiant le message par défault de l'exception InvalidTokenError avec une description par défaut : Invalid token qui peut être override par un message plus clair.

Reply:
token => accessToken corrigé

Reply:
Cette méthode `extractExternalUserFromIdToken`  est appelée uniquement par le use-case `create-user-and-reconcile-to-schooling-registration-from-external-user`, qui ne fait pas partie du scope du ticket. Comme vu ensemble, pas de BSR > 10 minutes sur ce ticket, ce sera fait dans un ticket dédié.